src/pages/workspace/accounting/qbo/advanced/QuickbooksAdvancedPage.tsx

```diff
@@ -0,0 +1,143 @@
+            isActive: Boolean(autoCreateVendor),
+            onToggle: () => Policy.updatePolicyConnectionConfig(policyID, CONST.QUICK_BOOKS_CONFIG.AUTO_CREATE_VENDOR, !autoCreateVendor),
+            pendingAction: pendingFields?.autoCreateVendor,
+        },
+    ];
+
+    return (
+        <AdminPolicyAccessOrNotFoundWrapper policyID={policyID}>
+            <PaidPolicyAccessOrNotFoundWrapper policyID={policyID}>
+                <FeatureEnabledAccessOrNotFoundWrapper
+                    policyID={policyID}
+                    featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
+                >
+                    <ScreenWrapper
+                        includeSafeAreaPaddingBottom={false}
+                        shouldEnableMaxHeight
+                        testID={QuickbooksAdvancedPage.displayName}
+                    >
+                        <HeaderWithBackButton title={translate('workspace.qbo.advancedConfig.advanced')} />
+
+                        <ScrollView contentContainerStyle={[styles.pb2, styles.ph5]}>
+                            {qboSyncToggleSettings.map((item) => (
+                                <ToggleSettingOptionRow
+                                    key={item.title}
+                                    title={item.title}
+                                    subtitle={item.subtitle}
+                                    shouldPlaceSubtitleBelowSwitch
+                                    wrapperStyle={styles.mv3}
+                                    isActive={item.isActive}
+                                    onToggle={item.onToggle}
+                                    pendingAction={item.pendingAction}
+                                />
+                            ))}
+
+                            <View style={styles.mv3}>
+                                <SpacerView
+                                    shouldShow
+                                    style={[styles.chatItemComposeBoxColor]}
+                                />
+                            </View>
+
+                            <ToggleSettingOptionRow
+                                title={translate('workspace.qbo.advancedConfig.reimbursedReports')}
+                                subtitle={translate('workspace.qbo.advancedConfig.reimbursedReportsDescription')}
+                                shouldPlaceSubtitleBelowSwitch
+                                wrapperStyle={styles.mv3}
+                                // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+                                pendingAction={pendingFields?.reimbursementAccountID || pendingFields?.collectionAccountID}
+                                isActive={Boolean(reimbursementAccountID && collectionAccountID)}
+                                onToggle={() => Policy.updatePolicyConnectionConfig(policyID, CONST.QUICK_BOOKS_CONFIG.REIMBURSEMENT_ACCOUNT_ID, bankAccounts?.[0].id ?? '')}
```

Comment:
I think the id of the default account to be used is the first item of the combination of bank accounts and credit card accounts.

we just have to make sure to combine credit card accounts in case the array of bank accounts is empty

Reply:
[based off this part of the design doc](https://docs.google.com/document/d/1LLrOYRGEb_k_Z6dttcS99zlbCNNdpD1QlXAbFJYbR7g/edit#bookmark=id.flphzh4jxz2m)